packages/jetbrains-plugin/src/main/kotlin/com/mongodb/jbplugin/codeActions/impl/RunQueryCodeActionBridge.kt

```diff
@@ -124,14 +114,6 @@ internal object RunQueryCodeAction : MongoDbCodeAction {
         )
     }
 
-    private fun shouldShowRunGutterIcon(node: Node<PsiElement>): Boolean {
```

Comment:
I removed this method because it was legacy, for when an invalid query would get to here, but now this doesn't happen anymore and we can now generate aggregates.